src/proto/Ethereum.proto

```diff
@@ -89,7 +89,10 @@ enum TransactionMode {
     Legacy = 0;
 
     // Enveloped transaction EIP2718 (with type 0x2), fee is according to EIP1559 (base fee, inclusion fee, ...) 
-    Enveloped = 1; 
+    Enveloped = 1;
+
+    // EIP4337-compatible UserOperation
+    EIP4337 = 2;
```

Comment:
no need to add this mode

Reply:
why? I need it to build a TransactionEip4337 based on additional properties

Reply:
I guess you can infer that it's EIP4337 from a `UserOperation` message?

Reply:
good point!

Reply:
No, this might not work actually. I also need my own implementations of `preHash`, `serialize` and `encode`
https://github.com/trustwallet/wallet-core/pull/2891/files#diff-ee5abff8d6706ab34d606ef7cfd4679e6da47d2bbb909fd9bf607981c6ed365aR221

Reply:
Cool, let's name it `UserOperation` then